cli/src/print.rs

```diff
@@ -90,3 +91,21 @@ pub fn print_sc_help(app: &mut Command, subcommand: &str) {
     }
     println!();
 }
+
+/// Limit a string to a specific length, using an ellipsis to indicate truncation.
+pub fn truncate(text: &str, max_length: usize) -> Cow<str> {
```

Comment:
I've decided to put this in the `print` module since it's directly related to printing things on the terminal. While I could have put it in `utils`, I'm generally not a big fan of a generic `utils` module so I did not want to further contribute to its existence.

Reply:
I agree. And I'd support a PR to move `utils::table_format` to `print::table_format` and delete `utils.rs`